--- a/colmet/common/backends/zeromq.py
+++ b/colmet/common/backends/zeromq.py
@@ -13,12 +13,8 @@

 from colmet.common.backends.base import InputBaseBackend, OutputBaseBackend
 
-try:
-    _snd_hwm = zmq.HWM
-    _rcv_hwm = zmq.HWM
-except AttributeError:
-    _snd_hwm = zmq.SNDHWM
-    _rcv_hwm = zmq.RCVHWM
+_snd_hwm = zmq.SNDHWM
+_rcv_hwm = zmq.RCVHWM
 
 
 class ZMQInputBackend(InputBaseBackend):
