--- a/colmet/common/metrics/procstats.py
+++ b/colmet/common/metrics/procstats.py
@@ -53,8 +53,8 @@
                                 'meminfo_vmallocused'),
         'meminfo_vmallocchunk': (UInt64(), 'kbytes', 'none',
                                  'meminfo_vmallocchunk'),
-        'meminfo_hardwarecorrupted': (UInt64(), 'kbytes', 'none',
-                                      'meminfo_hardwarecorrupted'),
+        #'meminfo_hardwarecorrupted': (UInt64(), 'kbytes', 'none',
+        #                              'meminfo_hardwarecorrupted'),
         'meminfo_anonhugepages': (UInt64(), 'kbytes', 'none',
                                   'meminfo_anonhugepages'),
         'meminfo_hugepages_total': (UInt64(), 'count', 'none',

--- a/colmet/common/backends/zeromq.py
+++ b/colmet/common/backends/zeromq.py
@@ -13,12 +13,8 @@

 from colmet.common.backends.base import InputBaseBackend, OutputBaseBackend
 
-try:
-    _snd_hwm = zmq.HWM
-    _rcv_hwm = zmq.HWM
-except AttributeError:
-    _snd_hwm = zmq.SNDHWM
-    _rcv_hwm = zmq.RCVHWM
+_snd_hwm = zmq.SNDHWM
+_rcv_hwm = zmq.RCVHWM
 
 
 class ZMQInputBackend(InputBaseBackend):
@@ -69,7 +69,8 @@
         self.context = zmq.Context()
         self.hostname = socket.gethostname()
         self.socket = self.context.socket(zmq.PUSH)
         self.socket.setsockopt(zmq.LINGER, self.options.zeromq_linger)
+        print(f'{_snd_hwm} / {self.options.zeromq_hwm}')
         self.socket.setsockopt(_snd_hwm, self.options.zeromq_hwm)
         self.socket.connect(self.options.zeromq_uri)
         LOG.debug("Use the URI '%s'" % self.options.zeromq_uri)
